addform.component.html

```diff
@@ -1,7 +1,7 @@
 <form name="addForm" style="overflow: hidden;" onload="reset();">
     <div style="padding-top: 3%;
     padding-right: 1%;
-    padding-left: 4%;">
+    padding-left: 40%;">
```

Comment:
the padding is too much please reduce it ...

Reply:
ok..understood